ajax/libs/drift-zoom/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "drift-zoom",
+  "description": "Easily add \"zoom on hover\" functionality to your site's images. Lightweight, no-dependency JavaScript.",
+  "filename": "lib/Drift.js",
```

Comment:
This directory/file does not exist within the `dist` directory. I would assume the intention was for this to be `Drift.js`. As the source repo for this PR has been removed, I'll close this PR as nothing more can be done here. Please open a new PR if you wish to still add this library.